modules/rpc/handler.go

```diff
@@ -30,6 +30,8 @@ import (
 	"go.uber.org/yarpc/api/transport"
 )
 
+const panicResponse = "Server Error"
```

Comment:
two things:

1. This var looks familiar from another diff
2. Underscore the const variables, please